terminal/partials/page-content/revision.html

```diff
@@ -31,11 +31,14 @@
             {%- if config.repo_name == "Bitbucket" -%}
             See revision history on <a href="{{ page.edit_url | replace('mode=edit','mode=read') }}">{{ config.repo_name }}</a>.
             {%- endif -%}
+            {%- if config.repo_name == "Codeberg" -%}
+            See revision history on <a href="{{ page.edit_url | replace('_edit','commits') }}">{{ config.repo_name }}</a>.
```

Comment:
Yes I thought the same thing, I would actually be in favour of having `branch` specified in case the redirect changes some time in the future. I'll edit the path

Reply:
awesome
ty!